src/test/scala/com/campudus/tableaux/CreationTest.scala

```diff
@@ -118,4 +118,72 @@ class CreationTest extends TableauxTestBase {
       assertEquals(expectedJson1, test)
     }
   }
-}
+
+  @Test
+  def createCompleteTable(): Unit = okTest {
+    val createCompleteTableJson = Json.obj(
+      "tableName" -> "Test Nr. 1",
+      "cols" -> Json.obj(
```

Comment:
Ok, so will it be `cols : { type : <array>, names : <array> }` now? Did you change that?